submissions/Zakkarat/memory-pair-game/stylesheet/main.css

```diff
@@ -0,0 +1,116 @@
+body {
+  margin: 0;
+  background-repeat: no-repeat;
+  background-image: url(../img/back.jpg );
+  background-size: cover;
+  height: 100%;
+}
+html {
+  height: 100%;
+}
+.container {
+  display: flex;
+  width: 910px;
+  height: 626;
+  justify-content:space-around;
+  flex-wrap: wrap;
+  min-height: 58%;
+  background-color: #032333;
+  margin: auto;
+  border-radius: 5px;
+  box-shadow: 8px 5px 16px 3px;
+}
+
+.flip-container {
+  margin-top: 28px;
+  margin-bottom: 10px;
+	perspective: 1000px;
+  margin-left: 2%;
+  margin-right: 2%;
+}
+
+	.toFlip{
+		transform: rotateY(180deg);
+	}
+
+.flip-container, .front, .back, img {
+	width: 117px;
+	height: 249px;
+}
+
+
+.flipper {
+	transition: 0.5s;
+	transform-style: preserve-3d;
+	position: relative;
+}
+
+
+.front, .back {
+	backface-visibility: hidden;
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+
+
+.front {
+	z-index: 2;
+	transform: rotateY(0deg);
+}
+
+.back {
+	transform: rotateY(180deg);
```

Comment:
Hmmm. Why do you have different indentation in the same file?

Reply:
It's strange, but in editor it looks good. Now everything cool!

Reply:
Actually I see the mess again. Can you post me a picture how you see your css file here in github?

Reply:
[Here it is](https://ibb.co/rx9Wfhv), used CSS comb for prettifying